french/feu-de-camp.md

```diff
@@ -20,10 +20,12 @@ Que faites-vous ?
 
 [Je dessine une porte avec un syphon de crème chantilly qui traine](dessine/porte.md)
 
-[Je ne parle pas francais](../english/marshmallow.md)
+[I don't speak French](../english/marshmallow.md)
 
 [J'appelle ma mère immédiatement](appel-maman/appeler-maman.md)
 
-[Je demande de l'aide a Google](google/google.md)
+[Je demande de l'aide à Google](google/google.md)
 
 [J'appelle le genie](appelle-genie/appelle-genie.md)
+
+[Je fonce dans le mur pour le défoncer](frapper-le-mur.md)

```

Comment:
It looks like this is a new link you added, but there is no corresponding new page. Did you create a new page but forget to add it to Git? If not, please go ahead and create one, since I don't want the story to have dead links in it.